config.py

```diff
@@ -373,10 +373,12 @@ class configuration:
                               ('DOUBLE', None),
                               ('NVCC', None),
                               ('CCACHE_CC', 'nvcc')]
-  # define OPENMP
+
+  # define OPENMP and SWIG (for log output)
   for compiler in macros:
     for precision in macros[compiler]:
       macros[compiler][precision].append(('OPENMP', None))
+      macros[compiler][precision].append(('__SWIG__', None))
```

Comment:
I was taught to use __WHATEVER__ so that it doesn't conflict with any names you might actually use. But I can change this to SWIG to fit the OpenMOC style.